projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/model/JobDeployment.java

```diff
@@ -29,6 +29,8 @@ public class JobDeployment {
 
   private String cronJobName;
 
+  private String schedule;
```

Comment:
what is this field for?

Reply:
This is the cron schedule. The class didn't have this field which is required in the new deployment entity, so we should add it here - it's used when merging deployments.

Reply:
but it will always be empty?

Reply:
the field is automatically populated.